### ISSUE TITLE
OpenSSH 7.8 causes VMware Workstation and Player networking SSH bug on Linux VMs

### ISSUE THREAD

**Issue by hermidalc** [vmware/open-vm-tools#287]
I found it on my Fedora 28 VM after applying updates this weekend. I read others have same problem on other distros as well.  Looks like a VMware bug.  See the details here 
https://bugzilla.redhat.com/show_bug.cgi?id=1624437.  I also found another report on OpenSSH of same issue for another Linux distro VM on Workstation or Player.  https://groups.google.com/forum/m/#!msg/opensshunixdev/5FK67SCpPg8/Y_QsxNkQBAAJ

I’m running latest Workstation 14.1.3.

**Comment by jstermask** [comment]
I confirm. I upgraded a Manjaro distribution. openssh is now 7.8p1-1. This Distribution runs on VMWare Player 14.1.3.
I have always had ssh packet_write_failed Broken Pipe error since then.

**Comment by hermidalc** [comment]
@jstermask and others - the workaround posted on RedHat Bugzilla works for me, in `~/.ssh/config`:
```bash
Host *
    IPQoS lowdelay throughput
```
of course the other workaround also works:
```bash
sudo dnf downgrade openssh-clients-7.7p1
```

**Comment by zack** [comment]
Many thanks! Been unable to figure this out for a little while now. Lots of errors out there that look like this but aren't this. That fix worked like a charm, @hermidalc.

Just for search engine users in the future, since my error output seemed somewhat unique:

```
packet_write_wait: Connection to 192.30.253.112 port 22: Broken pipe
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by hermidalc** [comment]
To the open-vm-tools team, I've been running the new VMware Workstation 15.0 for a couple days now and,  not sure if this issue is on the tools side or the workstation side, but the issue persists with 15.0.  If I comment out the fix you get the same SSH problems.

**Comment by johnyoonh** [comment]
The same issue for me on VM Workstation 15.0 Mojave. I've tried different openssl but I cannot get openssh to go below 7.8p1. 7.9p1 didn't help either.

**Comment by n6iap** [comment]
Running Fedora 28 and - the workaround posted on RedHat Bugzilla works for me.  Created file:
    ~/.ssh/config:
#Added lines to fix.
Host *
    IPQoS lowdelay throughput
#end of new file edit

**Comment by hermidalc** [comment]
On VMware Workstation for Windows 15.0.2 host, Fedora 29 guest, open-vm-tools 10.3.0 the problem still persists and requires the ssh config workaround.

**Comment by altmas5** [comment]
Same problem here 
I'm using VMWare Workstation Pro 15.0.2 running 
Mandriva Linux release 2006.0 (Official) for i586
Kernel 2.6.12-12mdksmp on an i686

**Comment by knuckolls** [comment]
I encountered this on FreeBSD-12-STABLE with a fresh version of workstation pro with a freshly compiled port of git. git clone would fail with a broken pipe. This is the only thing that fixed it. Hopefully this is a good breadcrumb for someone else.

$ cat .ssh/config
Host *
  IPQoS throughput

**Comment by kernelsmith** [comment]
It would be so neat if someone from VMware actually acknowledged this bug either here or on the VMware community forums... maybe, ya know, let users know they were even aware.

**Comment by dsouders** [comment]
Thank you for calling this to our attention.  I have filed an internal bug to track the issue.

**Comment by kernelsmith** [comment]
@dsouders thanks for responding. FYI:

**Comment by Vivalio** [comment]
Can confirm there is still an issue here, any status on the fix or are we stuck with he work around?

**Comment by cicku** [comment]
Yeah I still can confirm this on Fedora 29 + VM 15.0.2 + openssh-7.9p1-4.fc29

The workaround solves the problem.

**Comment by stanguturi** [comment]
> Yeah I still can confirm this on Fedora 29 + VM 15.0.2 + openssh-7.9p1-4.fc29

The issue was fixed in VMware workstation 15.1 Please upgrade the workstation version and re-open if the issue still happens.
